core/src/main/scala/stryker4s/config/Config.scala

```diff
@@ -12,7 +12,8 @@ case class Config(mutate: Seq[String] = Seq("**/main/scala/**/*.scala"),
                   reporters: List[MutantRunReporter] = List(new ConsoleReporter),
                   logLevel: Level = Level.INFO,
                   files: Option[Seq[String]] = None,
-                  excludedMutations: Exclusions = Exclusions(Set.empty)) {
+                  excludedMutations: Exclusions = Exclusions(Set.empty),
+                  thresholds: Option[Thresholds] = None) {
```

Comment:
Why is this an option? This could also just be a `TreshHolds()` and putt the default break in the case class to 0.